### ISSUE TITLE
Cell editor does not open for double tap on mobiles, it opens fot quick 3 taps 

### ISSUE THREAD

**Issue by AMBudnik** [handsontable/handsontable#6961]
### Description

Cell editor does not open for double tap, we need to make quick 3 taps. It might be related to the fact that double-tap suppose to zoom the viewport. It's quite hard to do a quick 3 taps so this has to be changed.

### Steps to reproduce

1. Use a mobile device to open cell editor on any demo.

**Reproduction for double-tap**
![mobile](https://user-images.githubusercontent.com/10757813/83251756-61782f00-a1aa-11ea-855f-4af6d228375e.gif)

### Demo

https://jsfiddle.net/AMBudnik/8dehxb3z/5/ 7.4.2 (broken)
https://jsfiddle.net/AMBudnik/8dehxb3z/ 8.0.0-beta2-rev10 (broken)

### Your environment
* Handsontable version: 7.4.2
* Browser Name and version: Chrome for iOS
* Operating System: iOS 13.5 on iPhone Xs

**Comment by nikersify** [comment]
Closed in #7470

**Comment by AMBudnik** [comment]
This issue is closed temporarily. It will be reopened for development as it became a part of **Interoperability problems using Mobile devices and some core features** task reported at https://github.com/handsontable/handsontable/issues/7520

I will make sure to inform everyone interested in this topic after the official fix.

Please feel welcome to makes any comments on this issue. And if you experience similar behavior feel free to contact me at support@handsontable.com

**Comment by aninde** [comment]
Opening editor after 3 click can still be reproduced on Chrome / iOS 14.4 on our [demo](https://handsontable.com/examples?manual-resize&manual-move&conditional-formatting&context-menu&filters&dropdown-menu&headers).

**Comment by Osky772** [comment]
I've tried to fix the double-tap problem for iOS on Chrome browser in https://github.com/handsontable/handsontable/pull/7677 but unfortunately, I've failed. What I discovered though, is that on iOS on Chrome there is no possibility to fast double-tap on Handsontable - the second event isn't captured in https://github.com/handsontable/handsontable/blob/da99d607675f332c0d83c87be816dbe6cd3eb9f1/src/3rdparty/walkontable/src/event.js#L294-L301
There is no problem on Safari, the problem is only on the Chrome browser. The problem is Handsontable-related, because, on any DOM element, the second event is captured.
